routes/testuser.router.js

```diff
@@ -1,26 +1,27 @@
 const express = require('express');
 const router = express.Router();
-const {extend} = require("lodash")
+const { extend } = require("lodash")
 
 const { TestUser } = require('../models/user.model');
 
 
+
 router.route("/:userId/product/:productId")
-.post(async (req, res) => {
- try{
-  const {userId, productId} = req.params
-  console.log(userId, productId)
-  
-  const updatedCart =await User.findByIdAndUpdate(userId, { $push: { cart: productId}}).populate("cart wishlist")
-  // updatedCart = extend (updatedCart, {quantity : 1})
-  console.log(updatedCart)
-  // updatedCart.save();
+  .post(async (req, res) => {
+    try {
+      const { userId, productId } = req.params
+      console.log(userId, productId)
+
+      const updatedCart = await User.findByIdAndUpdate(userId, { $push: { cart: productId } }).populate("cart wishlist")
+      // updatedCart = extend (updatedCart, {quantity : 1})
```

Comment:
At many places there is commented code, remove it so that your code will look clean.